views/auth/store.ejs

```diff
@@ -0,0 +1,183 @@
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta
+            name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- define links -->
+    <link rel="stylesheet" href="/css/store.css">
+    <!-- import Bootstrap 5-->
+    <link rel="stylesheet" href="/sources/bootstrap.css">
+    <title>Store</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css">
+
+</head>
+<body>
+<div class="container">
+    <div class="row">
+        <div class="col-sm-5 ">
+            <select name="store" id="store">
+
+                <option value="Bakım">Araç Bakım</option>
+                <option value="Motor">Motor Tamir</option>
+                <option value="Parça">Araç Parça</option>
+                <option value="Modifiye">Araç Modifiye</option>
+                <option value="Kaporta">Kaporta Tamir</option>
+            </select>
+
+
+        </div>
+        <div class="col-sm-7 d-flex justify-content-end">
+            <button class="kare"><i class="fas fa-th-large"></i></button>
+            <button class="filter"><i class="fas fa-filter"></i></button>
+            <select name="arac" id="arac">
+                <option class="secenek" value="Yeni">Yeni Mağazalar</option>
+                <option class="secenek" value="yüksek">Yüksek Puanlılar</option>
+                <option class="secenek" value="a'z">A'dan Z'ye</option>
+                <option class="secenek" value="z'a">Z'den A'ya</option>
+                <option class="secenek " value="favori">Favori Mağazam</option>
+            </select>
+
+
+        </div>
+        <h1 class="d-flex justify-content-center ">Mağazalar</h1>
+    </div>
+</div>
+
+<div class="card m-5">
+    <div class="row g-2">
+        <div class="col-md-4">
+            <img class="card-img-top"
+                 src="https://lh3.googleusercontent.com/hukMQ4A0AQpYz52IHl2M5Ppbbmmh71RwXjuIT6699o4ZPhadhwBMDmZSDDG5nX36xeXdL6Ui=w1080-h608-p-no-v0"
+                 alt="...">
+        </div>
+        <div class="col-md-7">
```

Comment:
grid sistemi 12 pirim ile çalışıyor, sen yukarı col-md-4 kullandın ve burda col-md-7 yani toplam 11 bunun için bu satırı col-md-8 değiştir.
ve hepsini aynı değer ver, sedece o değil